.vscode/settings.json

```diff
@@ -1,9 +1,45 @@
 {
     "editor.trimAutoWhitespace": false,
     "files.trimTrailingWhitespace": false,
+    "files.associations": {
+        "*.csproj": "msbuild",
```

Comment:
Can extensions declare file type associations like this? it might be good for Ionide/omnisharp to do these.

There's also an [MSBuild language service](https://marketplace.visualstudio.com/items?itemName=tintoy.msbuild-project-tools) I've enjoyed that may be useful for this devcontainer.

Reply:
Of course you did - I'm just unable to read apparently :D